assets/ui/statisticsScreen.ui

```diff
@@ -0,0 +1,289 @@
+                            "content": {
+                              "type": "migLayout",
+                              "id": "spadesTeamPlayerStatistics",
+                              "family": "playerNames",
+                              "rowConstraints": "0",
+                              "colConstraints": "[300][100][100]"
+                            }
+                          }
+                        ]
+                      }
+                    ]
+                  },
+                  {
+                    "type": "MigLayout",
+                    "rowConstraints": "0",
+                    "contents": [
+                      {
+                        "type": "UILabel",
+                        "id": "spadesTeamScore",
+                        "text": "Spades Score",
+                        "layoutInfo": {
+                          "cc": ""
+                        }
+                      },
+                      {
+                        "type": "UILabel",
+                        "text": "/",
+                        "layoutInfo": {
+                          "cc": ""
+                        }
+                      },
+                      {
+                        "type": "UILabel",
+                        "id": "spadesGoalScore",
+                        "text": "0",
+                        "layoutInfo": {
+                          "cc": ""
+                        }
+                      }
+                    ]
+                  },
+                  {
+                    "type": "MigLayout",
+                    "rowConstraints": "[30][150]",
+                    "layoutInfo": {
+                      "cc": "cell 0 2"
+                    },
+                    "contents": [
+                      {
+                        "type": "UIImage",
+                        "image": "LightAndShadowResources:heartsIcon"
+                      },
+                      {
+                        "type": "UILabel",
+                        "text": "Hearts Team",
+                        "family": "heartsTeamLabel"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
+                      {
+                        "type": "UISpace"
+                      },
```

Comment:
Why exactly do we need to do it like this?

Reply:
This seems like the easiest way to align the elements (labels) in the necessary layout, I don't think we would have the flexibility in arrangement otherwise.

Reply:
I'll need to have another look at the screen in-game, but this seems weird... on the other hand TeraNUI is kind of weird, so maybe it's just another "oh wow, it's _that_ weird" moment for me :sweat_smile:

Reply:
So _theoretically_, it should be possible to span elements as explained in the old [TeraNUI Tutorial Wiki](https://github.com/Terasology/TutorialNui/wiki/MigLayout#merging-and-splitting-cells). However, I gave it a quick try and it doesn't seem to work anymore... So I guess we need to stick with this for now.